web-app/js/portal/data/MetadataRecord.js

```diff
@@ -126,7 +126,10 @@ Portal.data.MetadataRecord = function() {
 
         var allowedOnlineResources = [];
 
-        Ext.each(Portal.app.appConfig.portal.metadataProtocols.webPage, function(protocol) {
+        Ext.each(Portal.app.appConfig.portal.metadataProtocols.supplementary, function(protocol) {
+            allowedOnlineResources.push(protocol.trim());
+        });
+        Ext.each(Portal.app.appConfig.portal.metadataProtocols.metadataRecord, function(protocol) {
```

Comment:
`metadataRecord` wasn't included before, why now?

Also, the two `Ext.each()` can probably be rolled in to one:

```javascript
Ext.each(Portal.app.appConfig.portal.metadataProtocols.supplementary.concat(Portal.app.appConfig.portal.metadataProtocols.metadataRecord), function(protocol) {
  allowedOnlineResources.push(protocol.trim());
});